packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/edit-display-name/controller.js

```diff
@@ -0,0 +1,62 @@
+export default class EditNameCtrl {
+  /* @ngInject */
+  constructor($translate, Nutanix) {
+    this.$translate = $translate;
+    this.Nutanix = Nutanix;
+  }
+
+  onUpdateDisplayNameFormSubmit() {
+    if (!this.actionEnabled) {
+      return this.goBack();
+    }
+
+    this.loading.update = true;
+
+    return this.Nutanix.updateDisplayName({
+      serviceId: this.nodeId,
+      serviceName: this.dedicatedServer.name,
+      displayName: this.displayName,
+    })
+      .then(() => {
+        this.goBack(
+          this.$translate.instant(
+            'nutanix_server_displaname_edit_displayname_success',
+          ),
+        );
+      })
+      .catch(({ data }) => {
+        this.goBack(
+          this.$translate.instant(
+            'nutanix_server_displaname_edit_displayname_error',
```

Comment:
```suggestion
            'nutanix_node_general_info_edit_display_name_error',
```
please use 'nutanix_node' prefix for all node related translations.